### ISSUE TITLE
1.2.1 can't input chinese

### ISSUE THREAD

**Issue by scloudyy** [Microsoft/vscode#7997]
- VSCode Version: 1.2.1
- OS Version: win7

Steps to Reproduce:

When using Chinese input method to type one word, just like '沈' which is composed by 'shen', but there will appear a lot of unexpected words.
This bug appears only after 1.2.1

![qq 20160622194309](https://cloud.githubusercontent.com/assets/15358886/16265443/fa9ef274-38b1-11e6-8d15-a7db86b8721a.png)

![qq 20160622194321](https://cloud.githubusercontent.com/assets/15358886/16265465/11d9e5d4-38b2-11e6-9fba-cdae045cb44c.png)

**Comment by Emeryao** [comment]
maybe it's related to your input method.
可以换个输入法试试兄弟，我输入中文是OK的

**Comment by rebornix** [comment]
Can not repro with default Chinese IME

**Comment by Emeryao** [comment]
@scloudyy 
Then the most likely reason is your input method.
个人认为：搜狗输入法之类的流氓软件还是不用的好

**Comment by Yukaii** [comment]
Same issue here with build-in Zhuyin IM(Traditional Chinese) on OS X El Capitan：
![screen shot 2016-06-26 at 6 27 14 pm](https://cloud.githubusercontent.com/assets/4230968/16361720/2dd84b00-3bcc-11e6-8387-dc77cf522174.png)

**Comment by Elderry** [comment]
Reproduced this problem with default Chinese IME(Microsoft Pinyin), version 1.2.1 on Windows 10.

**Comment by alexandrudima** [comment]
Assigning to June for investigation, I am not aware of any recent changes in this area.

@scloudyy @Emeryao @Yukaii @Elderry 
Can you please help me out and write in here a comment that I could bring to https://github.com/Microsoft/vscode/wiki/IME-Test

I would very much like to bring this use-case to our IME Test as two items, once for Windows and once for Mac, but I would need it to be in a format similar to the ones existing:

* **very very detailed** and clear steps such that devs unfamiliar with the specific input method can reproduce

* a gif of a good software showing how the behaviour should be. I used word, but if you don't have that, then please use any wide software that is doing it correctly

**Comment by Yukaii** [comment]
### [OS X 10.11.5]

- Add Zhuyin Input Method

  - System Preferences > Keyboard > Input Sources > Add > Chinese, Tranditional > Zhuyin

  ![](http://i.imgur.com/qlOlqlv.png)

- Focus VS Code and switch OS input method to `Zhuyin` (press Command + space)

  ![](http://i.imgur.com/CRGGrj4.png)


- Type `su3cl3` and press Enter. Here is how World looks like doing this (It should be "你好")

  ![](http://i.imgur.com/pqYKO6q.gif)

  ​

**Comment by alexandrudima** [comment]
@Yukaii Thank you! That is brilliant, I have added the test to https://github.com/Microsoft/vscode/wiki/IME-Test

**Comment by Emeryao** [comment]
@alexandrudima 
* Add the Chinese (Simplified) input method
  * Settings > Time & Language > Region & language > Add a language > Chinese (Simplified)
* Focus VS Code and switch OS input method to Chinese and choose Chinese mode and make sure the input method's name is `Microsoft Pinyin`:  
![chinese-ime](https://github.com/Microsoft/vscode/wiki/images/ime/windows-chinese-ime.png)  
* Type `ni` press Space and then `hao` and press Space. They should be typed in immediately. Here is how Word looks like doing this:
![1](https://cloud.githubusercontent.com/assets/3591899/16427212/d789af44-3d9e-11e6-87bd-aed8cdad15e2.gif)

**Comment by alexandrudima** [comment]
@Emeryao Thank you! I have added it to the wiki - https://github.com/Microsoft/vscode/wiki/IME-Test#windows-chinese-test-2

**Comment by erica-zhou** [comment]
Same issue here with build-in Pinyin IM on OS X El Capitan.

it ok when Input a Chinese word,but can't Input continuously

continuously input
![fc8a6938-dec0-4e8c-ae1d-2e84f5489459](https://cloud.githubusercontent.com/assets/8351089/16517244/1a308f9a-3faf-11e6-8fab-599e45e91bc9.png)

single word
![qq20160701-0](https://cloud.githubusercontent.com/assets/8351089/16517303/4ff11618-3faf-11e6-8b3f-96f262aada50.png)

**Comment by alexandrudima** [comment]
I have just tested on Windows and the test provided by @Emeryao works for me:

![svxjsk7y1v](https://cloud.githubusercontent.com/assets/5047891/16522428/063d2dd8-3f9e-11e6-981e-a05756ac8b8b.gif)

I believe this might be a mac specific issue.

**Comment by scloudyy** [comment]
I find this issue is not belong to VSC, it is a bug of amVim. if uninstall amVim, this issue will disappear. @Yukaii  @Elderry  @alexandrudima 
I have submit an new issue to amVim https://github.com/aioutecism/amVim-for-VSCode/issues/109

Maybe next time besides "VSCode Version" and "OS Version" we should also let others know what packages we have installled. @alexandrudima

**Comment by alexandrudima** [comment]
In this case, thank you for the IME tests, they are now in the wiki :+1:, and let's track further the problems with vim extensions + IME in #8133
